SLBperformance/DecodedSLBAnalysis.cc

```diff
@@ -512,7 +525,7 @@ for (Long64_t jentry=0; jentry<nentries;jentry++) {
       cdhisto[ilayer]->cd();
       
       for(int ichip=0; ichip<16; ichip++) {
-	if( cov.at(isca).at(ilayer).at(ichip)->GetEntries()>400) {
```

Comment:
Is it on purpose to always replace the 400 by 10, or should this depend on `frominjection`?